daisy_workflows/image_build/enterprise_linux/kickstart/rhel7-sap-post.cfg

```diff
@@ -31,5 +29,6 @@ unar
 "
 
 yum install -y ${SAP_PKGS}
-yum install -y fence-agents-gce resource-agents-gcp
+yum install -y fence-agents-gce resource-agents-gcp || exit 0
```

Comment:
the others use return, this uses exit

Reply:
Artifact of a failed experiment- changed.